Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@Cameron-Micka following our principle of "if it's in core SDK then it should be documented and tested", since this doesn't have documentation or tests and I'm not sure you have time to add those, could you please put this in MRTK.SDK/Experimental instead?

Something like:

MRTK.SDK/Experimental/Solvers/HandConstratintPalmUp.cs

Additionally, we might want to move the example scene to 

MRTK.Examples/Experimental/HandTracking/Scenes/HandUIExample.unity